tests/test_chat_reactions.py

```
@@ -0,0 +1,63 @@
+import pytest
+from unittest.mock import AsyncMock, MagicMock
+
+from tinyagentos.chat.reactions import maybe_trigger_semantic, WantsReplyRegistry
+
+
+@pytest.mark.asyncio
+async def test_regenerate_triggered_for_thumbs_down_on_agent_reply():
+    bridge = MagicMock(); bridge.enqueue_user_message = AsyncMock()
```

Ruff (0.15.10)

[error] Multiple statements on one line (semicolon)

(E702)